zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,35 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use std::time::Duration;
+
+use zebra_chain::{amount::COIN, block::Height};
+
+/// SlowStartInterval
+pub const SLOW_START_INTERVAL: Height = Height(20_000);
+
+// MaxBlockSubsidy
+pub const MAX_BLOCK_SUBSIDY: i64 = (12.5 * (COIN as f64)) as i64;
```

Comment:
```suggestion
/// The largest block subsidy, used before the first halving.
///
/// We use `25 / 2` instead of `12.5`, so that we can calculate the correct value without using floating-point.
/// This calculation is exact, because COIN is divisible by 2, and the division is done last.
pub const MAX_BLOCK_SUBSIDY: i64 = (25 * COIN) / 2;
```

Reply:
Changed to this at https://github.com/ZcashFoundation/zebra/pull/1051/commits/5f0cf2964765b3b5fb4431ae8b9ff540f5338722